bubble-wrap-style.yaml

```diff
@@ -4652,45 +4758,45 @@ layers:
                 visible: false
             polygons:
                 order: global.feature_order
-                #debug for polygon merging
-                #order: |
-#                   function() {
-#                       if( feature.id ) {
-#                           return 1001;
-#                       } else {
-#                           return 1000;
-#                       }
-#                   }
-                #debug for polygon merging
-                #color: |
-#                   function() {
-#                       if( feature.id ) {
-#                           return [
-#                               0.7 * (parseInt(feature.id, 16) / 100 % 1),     // jshint ignore:line
-#                               0.7 * (parseInt(feature.id, 16) / 10000 % 1),   // jshint ignore:line
-#                               0.7 * (parseInt(feature.id, 16) / 1000000 % 1), // jshint ignore:line
-#                               1
-#                           ];
-#                       } else {
-#                           if( feature.scale_rank == 6 ) {
-#                               return [1.,0.,0.,1];
-#                           } else if( feature.scale_rank == 5 ) {
-#                               return [0.,1.,0.,1];
-#                           } else if( feature.scale_rank == 4 ) {
-#                               return [0.,0.,1.,1];
-#                           } else if( feature.scale_rank == 3 ) {
-#                               return [0.,0.,0.5,1];
-#                           } else if( feature.scale_rank == 2 ) {
-#                               return [0.,0.5,0.,1];
-#                           } else if( feature.scale_rank == 1 ) {
-#                               return [0.,0.5,0.,1];
-#                           } else if( feature.scale_rank == 0 ) {
-#                               return [0.5,0.,0.,1];
-#                           } else {
-#                               return [0.5,0.5,0.5,1];
-#                           }
-#                       }
-#                   }
+#                #debug for polygon merging
```

Comment:
This building section needs revisiting... I had a bad merge conflict along the way.

Reply:
Seems fine, actually.